package exceptions

object NoRAPercentFound : Throwable() {

}
